src/GitHub.Exports.Reactive/Collections/TrackingCollection.cs

```diff
@@ -0,0 +1,934 @@
+                    SwapIndex(list, i, 1, indexCache);
+                }
+
+            else // replacing element has higher sorting order, find the correct spot towards the end
+                for (var pos = i + 1; i < list.Count - 1 && comparer(list[i], list[pos]) > 0; i++, pos++)
+                {
+                    Swap(list, i, pos);
+                    SwapIndex(list, i, -1, indexCache);
+                }
+            indexCache[list[i]] = i;
+            return i;
+        }
+
+        /// <summary>
+        /// Swap two elements
+        /// </summary>
+        static void Swap(IList<T> list, int left, int right)
+        {
+            var l = list[left];
+            list[left] = list[right];
+            list[right] = l;
+        }
+
+        static void SwapIndex(IList<T> list, int pos, int change, Dictionary<T, int> cache)
+        {
+            if (cache.ContainsKey(list[pos]))
+                cache[list[pos]] += change;
+        }
+
+        static int BinarySearch(List<T> list, T item, Func<T, T, int> comparer)
+        {
+            return list.BinarySearch(item, new LambdaComparer<T>(comparer));
+        }
+
+#if DISABLE_REACTIVEUI
+        static IScheduler GetScheduler(IScheduler scheduler)
+        {
+            Dispatcher d = null;
+            if (scheduler == null)
+                d = Dispatcher.FromThread(Thread.CurrentThread);
+            return scheduler ?? (d != null ? new DispatcherScheduler(d) : null as IScheduler) ?? CurrentThreadScheduler.Instance;
+        }
+#endif
+
+        bool disposed = false;
+        void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                if (!disposed)
+                {
+                    disposed = true;
+                    queue = null;
+                    disposables.Dispose();
+                }
+            }
+        }
+
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        struct ActionData
+        {
+            readonly public TheAction TheAction;
+            readonly public int Position;
+            readonly public int OldPosition;
+            readonly public int Index;
+            readonly public int IndexPivot;
+            readonly public bool IsIncluded;
+            readonly public T Item;
+            readonly public T OldItem;
+            readonly public List<T> List;
+
+            public ActionData(ActionData other, int index, int indexPivot)
+            {
+                TheAction = other.TheAction;
+                Item = other.Item;
+                OldItem = other.OldItem;
+                Position = other.Position;
+                OldPosition = other.OldPosition;
+                List = other.List;
+                Index = index;
+                IndexPivot = indexPivot;
+                IsIncluded = other.IsIncluded;
+            }
+
+            public ActionData(ActionData other, int position)
```

Comment:
Did you remember to push those changes? The only change I see is a merge to master. The constructors haven't changed.